src/components/reusable/sideDrawer/sideDrawer.scss

```diff
@@ -0,0 +1,173 @@
+@use '../../../common/scss/global.scss';
+@use '~@kyndryl-design-system/shidoka-foundation/scss/mixins/typography.scss';
+
+:host {
+  display: inline-block;
+}
+
+dialog {
+  padding: 0;
+  color: inherit;
+  background: var(--kd-color-background-ui-default);
+  border: 1px solid var(--kd-color-border-light);
+  box-shadow: 0px 0px 40px 0px rgba(0, 0, 0, 0.12);
+  right: -100%;
+  min-height: 100%;
+  left: auto;
+  margin: 0;
+  //animation: slideOutRight 3s linear forwards;
+
+  &[open] {
+    animation: slideInRight 0.3s forwards;
```

Comment:
For the animations/transition, copy my changes made to Modal, seen here: https://github.com/kyndryl-design-system/shidoka-applications/pull/105

The slide animation should now be `400ms ease-out`.

Reply:
I think mine one is 400ms forwards.

Reply:
still facing issue while close the drawer.

Reply:
The modal updates on support close animation in Chromium as a progressive enhancement. Firefox/Safari will still only get open animation. You can test this on Modal in storybook.

Reply:
`ease-in/out` is just the animation curve. The actual states you're changing are the position using `transform: translateX()`

Reply:
Ex;
hidden:
```css
transform: translateX(100%); /* move 100% of the size to the right so it's off screen. */
```
visible:
```css
transform: translateX(0); /* move back on screen. */
```

Reply:
ok its done. Thanks for **allow-discrete** prop.

Reply:
done